index.md

```diff
@@ -96,19 +101,19 @@ This error often results in an **AttributeError** when you run your program.
 
 ### E1102: Not callable {#E1102}
 
-This error occurs when you try to call a value which is not a function.
-In the following example, we should not call `x()` because `x` refers to
-a value which is an integer, not a function.
+This error occurs when you try to call a value which is not a function, method, or callable object.
+In the following example, we should not call `x()` because `x` points to an integer, and calling an integer has no meaning.
```

Comment:
Please use the terminology "refers to" rather than "points to" (and fix this in places where this isn't so) -- this is the terminology typically used in CSC108/148.